Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/signinsignup/WelcomeScreen.kt

```diff
@@ -60,38 +59,33 @@ sealed class WelcomeEvent {
 
 @Composable
 fun WelcomeScreen(onEvent: (WelcomeEvent) -> Unit) {
-    var brandingBottom by remember { mutableStateOf(0f) }
     var showBranding by remember { mutableStateOf(true) }
-    var heightWithBranding by remember { mutableStateOf(0) }
 
-    val currentOffsetHolder = remember { mutableStateOf(0f) }
-    currentOffsetHolder.value = if (showBranding) 0f else -brandingBottom
-    val currentOffsetHolderDp =
-        with(LocalDensity.current) { currentOffsetHolder.value.toDp() }
-    val heightDp = with(LocalDensity.current) { heightWithBranding.toDp() }
     Surface(modifier = Modifier.supportWideScreen()) {
-        val offset by animateDpAsState(targetValue = currentOffsetHolderDp)
         Column(
             modifier = Modifier
                 .fillMaxWidth()
-                .brandingPreferredHeight(showBranding, heightDp)
```

Comment:
Nit: the `brandingPreferredHeight` modifier should no longer be needed with this approach